src/plugins/intel_gna/src/ops/util/util.hpp

```diff
@@ -195,7 +200,7 @@ static bool is_Tbit_fq(const std::shared_ptr<ngraph::Node>& node) {
     if (!fq_node)
         return false;
     auto levels = fq_node->get_levels();
-    return std::numeric_limits<T>::max() == levels;
+    return (std::numeric_limits<T>::max() == levels) || (std::numeric_limits<T>::max() == levels - 1);
```

Comment:
Looks like this condition should be fixed where we are processing FQ, here it looks strange

Reply:
> Looks like this condition should be fixed where we are processing FQ, here it looks strange

Here we check bit number after FQ. Not sure that we have to modify something in FQ processing in this PR.